.golangci.yaml

```diff
@@ -44,6 +45,192 @@ issues:
+        alias: clientcmd${1}
+      - pkg: k8s.io/client-go/tools/cache
+        alias: toolscache
+      - pkg: k8s.io/kubelet/config/v1beta1
+        alias: kubeletconfigv1beta1
+      - pkg: k8s.io/utils/clock/testing
+        alias: testclock
+      - pkg: k8s.io/apiserver/pkg/endpoints/openapi
+        alias: openapinamer
+      - pkg: k8s.io/apiserver/pkg/server
+        alias: genericapiserver
+      - pkg: k8s.io/apiserver/pkg/server/options
+        alias: genericoptions
+      - pkg: k8s.io/apiserver/pkg/server/storage
+        alias: serverstorage
+      - pkg: k8s.io/apiserver/pkg/util/feature
+        alias: utilfeature
+      - pkg: k8s.io/component-base/config
+        alias: componentbaseconfig
+      - pkg: k8s.io/component-base/logs/api/v1
+        alias: logsv1
+      - pkg: sigs.k8s.io/controller-runtime/pkg/client/fake
+        alias: fakeclient
+      - pkg: sigs.k8s.io/controller-runtime/pkg/log/zap
+        alias: logzap
+      - pkg: sigs.k8s.io/controller-runtime/pkg/log
+        alias: logf
+      - pkg: github.com/golang/mock/gomock
+        alias: gmock
+      # Gradener external imported packages
+      - pkg: github.com/gardener/hvpa-controller/api/v1alpha1
+        alias: hvpav1alpha1
+      - pkg: github.com/gardener/machine-controller-manager/pkg/apis/machine/v1alpha1
+        alias: machinev1alpha1
+      - pkg: github.com/gardener/etcd-druid/api/v1alpha1
+        alias: druidv1alpha1
+      # Gardener extension package
+      - pkg: github.com/gardener/gardener/extensions/.*/(\w+)/mock$
+        alias: extensionsmock${1}
+      - pkg: github.com/gardener/gardener/extensions/pkg/apis/config
+        alias: extensionsconfig
+      - pkg: github.com/gardener/gardener/extensions/pkg/controller
+        alias: extensionscontroller
+      - pkg: github.com/gardener/gardener/extensions/pkg/predicate
+        alias: extensionspredicate
+      - pkg: github.com/gardener/gardener/extensions/pkg/controller/([^m]\w+)
+        alias: extensions${1}controller
+      - pkg: github.com/gardener/gardener/extensions/pkg/controller/heartbeat/cmd
+        alias: extensionsheartbeatcmd
+      - pkg: github.com/gardener/gardener/extensions/pkg/controller/worker/helper
+        alias: extensionsworkerhelper
+      - pkg: github.com/gardener/gardener/extensions/pkg/util/secret/manager
+        alias: extensionssecretsmanager
+      - pkg: github.com/gardener/gardener/extensions/pkg/webhook
+        alias: extensionswebhook
+      - pkg: github.com/gardener/gardener/extensions/pkg/webhook/([^m]\w+)
+        alias:  extensions${1}webhook
+      # Gardener other packages
+      - pkg: github.com/gardener/gardener/pkg/api/extensions
+        alias: apiextensions
+      - pkg: github.com/gardener/gardener/pkg/apis/core
+        alias: gardencore
+      - pkg: github.com/gardener/gardener/pkg/apis/core/([\w\d]+)
+        alias: gardencore${1}
+      - pkg: github.com/gardener/gardener/pkg/apis/([^c]\w+)/([\w\d]+)
+        alias: $1$2
```

Comment:
I would propose to have `v1{alpha,beta}1helper` instead of `gardencorev1{alpha,beta}1helper`.